packages/cli/project.json

```
@@ -37,12 +37,18 @@
         ]
       }
     },
-    "test": {
+    "unit-test": {
       "executor": "@nx/vite:test",
       "outputs": ["{options.reportsDirectory}"],
       "options": {
-        "passWithNoTests": true,
-        "reportsDirectory": "../../coverage/cli"
+        "reportsDirectory": "../../coverage/cli/unit"
+      }
+    },
+    "integration-test": {
+      "executor": "@nx/vite:test",
+      "outputs": ["{options.reportsDirectory}"],
+      "options": {
+        "reportsDirectory": "../../coverage/cli/integration"
```

Maybe explicit folder names would be clearer?

```suggestion
        "reportsDirectory": "../../coverage/cli/unit-tests"
      }
    },
    "integration-test": {
      "executor": "@nx/vite:test",
      "outputs": ["{options.reportsDirectory}"],
      "options": {
        "reportsDirectory": "../../coverage/cli/integration-tests"
```

(Would apply to all `project.json`s, obviously.)

---

Makes sense. Done!